Formula/cyral-gimme-db-token.rb

```diff
@@ -0,0 +1,130 @@
+class CyralGimmeDbToken < Formula
+  include Language::Python::Virtualenv
+
+  desc "Eases using Cyral for SSO login to databases"
+  homepage "https://cyral.com/docs/repo-connect#cli-token-retriever-for-sso"
+  url "https://files.pythonhosted.org/packages/18/05/afa2028e15ac618cb643001f8b4ede33f803307d30e9133286761a27eff1/cyral-gimme-db-token-0.5.0.tar.gz"
+  sha256 "df75d7e74545a3ebb32cc96279d42180cfdfc44d5a4b9def15b477ebcd9fc8c1"
+  license "Apache-2.0"
```

Comment:
Where does this license come from?? I dont think this is even an open source project.

Reply:
propose to disable this formula, https://github.com/Homebrew/homebrew-core/pull/133982

Reply:
@chenrui333 Hold off removal please, @antoniomrfranco, at least at the time, was employed by `cyral.com` so may be able to provide more information for us here.